RELEASE-NOTES.txt

```diff
@@ -1,6 +1,6 @@
 15.1
 -----
- 
+ * Fixes issue on Notifications tab when two screens were drawn on top of each other
```

Comment:
@malinajirka Quick reminder to add priority indicators for the release notes. I am guessing this is low priority release note and it's likely to be asked about in the editorial review. (as to what exactly changed) If you would like to set a different priority, please let me know or add a comment to the editorial post. (when available)

Reply:
Ohh, I totally missed the "release notes priority" post. Thanks! ;)

Confirming, that this is a low priority;).